serial_protocol.md

```diff
@@ -0,0 +1,157 @@
+#Protocol:
```

Comment:
hmm ya, let me figure out syncing changes in git. I thought I had it but I don't think so.